src/garage/torch/algos/vpg.py

```diff
@@ -449,9 +496,10 @@ def process_samples(self, itr, paths):
             pad_to_last(path['rewards'], total_length=self.max_path_length)
             for path in paths
         ])
-        baselines = torch.stack([
-            pad_to_last(self._get_baselines(path),
-                        total_length=self.max_path_length) for path in paths
+        returns = torch.stack([
+            pad_to_last(path['returns'], total_length=self.max_path_length)
+            for path in paths
         ])
+        baselines = self._value_function.predict(obs)
```

Comment:
please remove the predict interface.

the intention of this PR is to remove the fit/predict interface and retrieve the VF predictions by calling `forward` on the VF within the same computation graph as the policy.

Reply:
fixed